JavaAssignment/src/purva/PurvaAssign3_q5_Weekdays.java

```diff
@@ -0,0 +1,51 @@
+/* Write a Java program that keeps a number from the user and 
+generates an integer between 1 and 7 and displays the name of the weekday. (use switch case) */ 
+
+package purva;
+
+import java.util.Scanner;
+
+public class PurvaAssign3_q5_Weekdays {
+	public static void main(String[] args) {
+		int answer = 0;
+		do {
```

Comment:
try to not write all logic in main method.
write separate method which will return answer.